basic/html-tag/html-tags.md

```diff
@@ -0,0 +1,178 @@
+slug: introduction-to-html-tags
+tags: html
+cover: https://cdn.hashnode.com/res/hashnode/image/upload/v1704720765233/wr1xLBKVV.png?auto=format
```

Comment:
These details need to be enclosed in --- tags. Otherwise it'll not render on blog 

---
slug: introduction-to-html-tags
tags: html
cover: https://cdn.hashnode.com/res/hashnode/image/upload/v1704720765233/wr1xLBKVV.png?auto=format
domain: 10xdev.codeparrot.ai
saveAsDraft: false
---

Reply:
all done sir